src/PostsViewHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace src;
+
+require_once 'src/Models/PostModel.php';
+
+class PostsViewHelper
+{
+
+    public function displayAllPosts(array $posts) :void
```

Comment:
You should make this method return a string, it's more flexible than just echoing it - it also means it can have unit tests!